odoo/tools/misc.py

```diff
@@ -1156,6 +1160,37 @@ def formatLang(env, value, digits=None, grouping=True, monetary=False, dp=False,
             res = '%s %s' % (currency_obj.symbol, res)
     return res
 
+def formatLangDate(env, value, lang_code=False, date_format=False):
```

Comment:
Naming convention is pretty universally snake_case. Also why not just e.g. `format_date`?